### DISCUSSION TITLE
Kubernetes-Native Development with Quarkus and Eclipse JKube

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#32548] [Quarkus Blog/Website]
# Kubernetes-Native Development with Quarkus and Eclipse JKube

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/kubernetes-native-development-with-quarkus-and-eclipse-jkube/

**Comment by jbride** [comment]
Excellent article.
Running  _rest-fights_ service on local machine in quarkus:dev mode while remainder of _super-heroes_ deployments ran in OpenShift worked perfectly.
Very powerful.

**Reply by edeandrea** [reply]
Thank you @jbride for the feedback!
